DuckDuckGo/NewTabPageViewController.swift

```diff
@@ -39,7 +39,7 @@ final class NewTabPageViewController: UIHostingController<NewTabPageView<Favorit
     private let shortcutsModel: ShortcutsModel
     private let shortcutsSettingsModel: NewTabPageShortcutsSettingsModel
     private let sectionsSettingsModel: NewTabPageSectionsSettingsModel
-    private let tab: Tab
+    private let _tab: Tab
```

Comment:
This can be renamed to whatever, I choose the simplest option, I'm open to change it

Reply:
@iccub We rarely use an underscore prefix for standalone properties like this, so what do you think about `associatedTab` instead?

Reply:
Renamed https://github.com/duckduckgo/iOS/pull/3299/commits/8d1f841e6782767cb4e336dc79371b679bd154da